### PR TITLE
update reference to example track composer

### PR DESCRIPTION
**Pull request by BlackCoyote**
also with my other commit giving usages apparently. I don't know why github added those commits to this pull request even after i sent this pull request  before commiting the rest..

### CODE REVIEW

File: src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java [hunk 1]
```diff
@@ -2152,7 +2170,8 @@ public void registerCoreMembers() {
         // @Tags
         // <e@entity.is_spawned>
         // @Usage
-        // TODO: Document Command Details
+        // - remove <player.location.find.entities.within[100].filter[is_mob.not]>
+        // Removes all entities within 100 blocks from the player, excluding players and NPCs.
```
Comment (mcmonkey4eva) [inline src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java]: I question this one.

File: src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java [hunk 2]
```diff
@@ -1937,7 +1948,11 @@ public void registerCoreMembers() {
         // @Tags
         // TODO: Document Command Details
         // @Usage
-        // TODO: Document Command Details
+        // - playeffect effect:drip_lava <player.location> qty:50 offset:0.3
+        // Plays particle effects at the player location that almost look like blood!
+        // @Usage
+        // - playeffect effect:red_dust qty:0 data:1 offset:<co@1,1,255.to_particle_offset>
+        // Plays the effect with an RGB value specified. effects taking RGB input: RED_DUST, SPELL, INSTANT_SPELL
```
Comment (mcmonkey4eva) [inline src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java]: All these usages are written backwards

File: src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java [hunk 3]
```diff
@@ -1443,11 +1443,17 @@ public void registerCoreMembers() {
         // @Author aufdemrand
         // @Group core
         // @Description
-        // TODO: Document Command Details
+        // Used to add certain scripts into the current script queue.
```
Comment (mcmonkey4eva) [inline src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java]: A one-liner is not an impressive complete description.

File: src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java (hunk 1 above)
Comment (Fortifier42) [inline src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java]: That will remove the player...
and any other players

**Pull request closed without merging**